### DISCUSSION TITLE
Crystal structure

### DISCUSSION THREAD

**Discussion by Sofia-Y** [neperfepx/neper#220] [Q&A] [answered]
Hi @rquey !

I appreciate your quick responses and presence on this platform. 

I have another simple question: I know I can specify the crystal symmetry using different descriptors provided in the documentation. However, I was wondering if Neper can also specify the crystal structure (FCC, BCC, HCP)?

Thanks in advance.

**Comment by rquey** [comment, marked as answer]
Neper does not take this information because it wouldn't do anything about it right now. It needs the crystal symmetry to compute disorientations, but no more.
